pkg/api/pod/util_patch.go

```diff
@@ -0,0 +1,26 @@
+package pod
```

Comment:
these two should cancel and both be able to be dropped:
```
UPSTREAM: Revert "UPSTREAM: <drop>: keep old pod available"
UPSTREAM: <drop>: keep old pod available
```

Reply:
Dropped